buildbot/buildbot_init.sh

```diff
@@ -97,6 +97,20 @@ userdel buildbot
 groupadd buildbot
 useradd buildbot -g buildbot -m -d /var/lib/buildbot
 
+if [[ "${HOSTNAME}" == ml-opt-dev* ]]
+then
+  echo "Building TFLite"
+  curdir="${PWD}"
```

Comment:
It might be cleaner to use `pushd /tmp/tflitebuild` instead  of `curdir="${PWD}"` followed by `cd /tmp/tflitebuild` and later do `popd` instead of `cd "${curdir}"`.

Reply:
ah yes! pushd. And I can pushd /tmp/tflitebuild / popd